index.bs

```diff
@@ -281,7 +281,8 @@ The API is defined by the following Web IDL fragment.
     interface WebAuthentication {
         Promise < ScopedCredentialInfo > makeCredential (
             Account                                 accountInformation,
-            sequence < ScopedCredentialParameters > cryptoParameters,
+            sequence < CredentialType >             supportedTypes,
+            sequence < AlgorithmIdentifier >        supportedAlgorithms,
```

Comment:
My experience from hanging out with RP types is that they would prefer to minimize their attack surface by carefully choosing a handful of combinations to support, while leaving older mechanisms around for compatibility (until they can be safely removed). I would not be surprised to see new algorithms to only be added to new credential types, for example. So a few years from now you'd very likely end up with something like [(originalType, RSA), (originalType, ECDSA), (newType, postQuantumAlg)].